### DISCUSSION TITLE
Issue Nr. 145 missing

### DISCUSSION THREAD

**Discussion by zwillikon** [mate-desktop/atril#617] [General]
Where is https://github.com/mate-desktop/atril/issues/145 gone?

I've found it through my search engine and it could possibly help me.

**Comment by lukefromdc** [comment]
That sounds like an issue at Github and isn't something I know how to help with

**Comment by lukefromdc** [comment]
https://github.com/mate-desktop/atril/issues?page=11&q=is%3Aissue

currently shows nothing between

https://github.com/mate-desktop/atril/issues/157
Inconsistent session restore behaviour

and

https://github.com/mate-desktop/atril/issues/146
"Atril without mate-desktop?"

These date back to 2015, nine years ago. No idea on my end if https://github.com/mate-desktop/atril/issues/145
ever existed or was lost in a server disk migration or something else

Note that there are normally many gaps in issue numbering on github.  Atril pull requests from that era similarily
jump from 144 to 147
